homeassistant/components/zha/const.py

```diff
@@ -6,3 +6,4 @@
 """
 # pylint: disable=W0614,W0401
 from .core.const import *  # noqa: F401,F403
+from .core.registries import *  # noqa: F401,F403
```

Comment:
Is there anything else using this file? I'd rather have one consts module.

Reply:
Nah, I’ll get the last few quirks remediated soon so I can kill this file.

Reply:
Gotcha